<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-M4N4JHAP" ignored="0" name="Addition" tests="10" failures="10" timestamp="2021-03-14T15:52:13 GMT" time="66.807" errors="0">
  <testcase name="AddAscendRangeAndDescendRange" time="6.96" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [80235.0] but found [95117.5000000001]">
      <![CDATA[java.lang.AssertionError: expected [80235.0] but found [95117.5000000001]
at Acacemy.additionTest.AddAscendRangeAndDescendRange(additionTest.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddAscendRangeAndDescendRange -->
  <testcase name="AddBigNumberandOne" time="5.096" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [8989899.0] but found [95117.5000000002]">
      <![CDATA[java.lang.AssertionError: expected [8989899.0] but found [95117.5000000002]
at Acacemy.additionTest.AddBigNumberandOne(additionTest.java:46)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddBigNumberandOne -->
  <testcase name="AddDecimalNumbers" time="5.499" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [5.34] but found [25.351000000099997]">
      <![CDATA[java.lang.AssertionError: expected [5.34] but found [25.351000000099997]
at Acacemy.additionTest.AddDecimalNumbers(additionTest.java:104)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddDecimalNumbers -->
  <testcase name="AddMorethanTwoNumbers" time="5.371" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [21.0] but found [131.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [21.0] but found [131.0000000001]
at Acacemy.additionTest.AddMorethanTwoNumbers(additionTest.java:83)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddMorethanTwoNumbers -->
  <testcase name="AddNegativeandPositiveNumbers" time="5.258" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [-7.0] but found [31.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [-7.0] but found [31.0000000001]
at Acacemy.additionTest.AddNegativeandPositiveNumbers(additionTest.java:97)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddNegativeandPositiveNumbers -->
  <testcase name="AddNmmbersgivesSevenDigitResult" time="5.733" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [5555555.0] but found [1.27727727725E10]">
      <![CDATA[java.lang.AssertionError: expected [5555555.0] but found [1.27727727725E10]
at Acacemy.additionTest.AddNmmbersgivesSevenDigitResult(additionTest.java:39)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddNmmbersgivesSevenDigitResult -->
  <testcase name="AddTwoBigNumbers" time="6.209" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [9.34934934E8] but found [6.23168592534E11]">
      <![CDATA[java.lang.AssertionError: expected [9.34934934E8] but found [6.23168592534E11]
at Acacemy.additionTest.AddTwoBigNumbers(additionTest.java:52)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddTwoBigNumbers -->
  <testcase name="AddTwoSingleDigits" time="5.247" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [8.0] but found [8.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [8.0] but found [8.0000000001]
at Acacemy.additionTest.AddTwoSingleDigits(additionTest.java:90)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddTwoSingleDigits -->
  <testcase name="AddZeroAndOne" time="5.202" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [1.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [1.0000000001]
at Acacemy.additionTest.AddZeroAndOne(additionTest.java:72)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddZeroAndOne -->
  <testcase name="AddZeroAndZero" time="5.177" classname="Acacemy.additionTest">
    <failure type="java.lang.AssertionError" message="expected [0] but found [1.0E-10]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [1.0E-10]
at Acacemy.additionTest.AddZeroAndZero(additionTest.java:60)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- AddZeroAndZero -->
</testsuite> <!-- Addition -->
